package.json

```diff
@@ -18,7 +18,8 @@
     "start": "node -r esm src/index.js",
     "lint": "eslint src --ext .js",
     "test": "jest",
-    "check-url": "node -r esm src/verifyUrl.js"
+    "check-url": "node -r esm src/verifyUrl.js",
```

Comment:
you can remove this one, it's was used to update url that have 301,  but it's no more used